integration/bwmf_test.go

```diff
@@ -0,0 +1,7 @@
+package integration
```

Comment:
should we leave this with the rest of bwmf so that when we move it, it is clear that this also need to be moved?